docs-ui/components/code.tsx

```diff
@@ -0,0 +1,154 @@
+import {withTheme} from '@emotion/react';
+import styled from '@emotion/styled';
+import copy from 'copy-text-to-clipboard';
+import Prism from 'prismjs';
+import 'prismjs/themes/prism.css';
+import {Component, createRef, Fragment, RefObject} from 'react';
+
+import {IconCode} from 'app/icons';
+import space from 'app/styles/space';
+import {Theme} from 'app/utils/theme';
+
+type Props = {
+	theme: Theme;
```

Comment:
Do you have prettier enabled? Looks like you're using tabs instead of spaces here. Prettier should automatically fix this for you.

Reply:
Yeah, but there's always some code editor conflicts with my other projects. Fixing this prompto

Reply:
Are you using VSCode? There should be some way to have configuration per project with it.

Reply:
Sublime Text. There's a plugin called EditorConfig that could fix this, but it requires adding a .editorconfig file to the project, which sounds annoying cause I have to ignore it for every commit

Reply:
You could always add it to your personal global gitignore :) https://medium.com/@peter_graham/how-to-create-a-local-gitignore-1b19f083492b

Reply:
woah learn something new every day 🌈